src/main/resources/edu/cornell/kfs/ksr/document/datadictionary/SecurityGroupMaintenanceDocument.xml

```diff
@@ -0,0 +1,82 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<beans xmlns="http://www.springframework.org/schema/beans" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:p="http://www.springframework.org/schema/p" xmlns:dd="http://rice.kuali.org/dd"
```

Comment:
This file has some references to Rice XML namespace URIs. I would suggest removing them.

Reply:
Good catch! Updated